suite-native/module-accounts-import/src/components/AccountImportLoader.tsx

```diff
@@ -1,41 +1,76 @@
-import React from 'react';
+/* eslint-disable global-require */
+import React, { useEffect } from 'react';
+import { Image } from 'react-native';
+import Animated, {
+    useAnimatedStyle,
+    withDelay,
+    withSequence,
+    withTiming,
+} from 'react-native-reanimated';
 
-import { Box, Text } from '@suite-native/atoms';
+import { Box, Loader, Text, VStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
-const fetchingStyle = prepareNativeStyle(() => ({
+type AccountImportLoaderProps = {
+    onLoad(): void;
+};
+
+const loaderContainerStyle = prepareNativeStyle(utils => ({
     flex: 1,
-    justifyContent: 'center',
+    justifyContent: 'space-between',
     alignItems: 'center',
+    marginTop: utils.spacings.medium * 10,
 }));
 
-const textStyle = prepareNativeStyle<{ opacity?: number }>((_, { opacity = 1.0 }) => ({
-    marginBottom: 13,
-    opacity,
+const imageStyle = prepareNativeStyle(() => ({ width: 283, height: 166 }));
+
+const textContainerStyle = prepareNativeStyle(utils => ({
+    overflow: 'hidden',
+    height: utils.typography.titleMedium.lineHeight,
 }));
+export const AccountImportLoader = ({ onLoad }: AccountImportLoaderProps) => {
+    const { applyStyle, utils } = useNativeStyles();
+    const { lineHeight } = utils.typography.titleMedium;
+    const lineVisibilityDuration = 1500;
```

Comment:
1. Move outside component. 
2. If constants value is just simple value (e.g. string / number, ...), we use uppercase SNAKE_CASE_NAMING